package.json

```diff
@@ -83,16 +84,16 @@
   "devDependencies": {
     "@storybook/addon-actions": "^4.1.1",
     "@storybook/addon-links": "^4.1.1",
-    "@storybook/react-native": "^4.1.1",
-    "babel-jest": "^23.6.0",
-    "babel-plugin-jest-hoist": "^23.2.0",
+    "@storybook/react-native": "^4.1.14",
+    "babel-jest": "^24.5.0",
+    "babel-plugin-jest-hoist": "^24.3.0",
```

Comment:
I updated these to try and fix another [issue](https://github.com/Skyscanner/backpack-react-native/pull/77/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.kt&file-filters%5B%5D=.snap#r265502545) but the upgrade doesn't seem to break anything so may as well do it.